k8s/deploy/xebialabs/application.yaml.tmpl

```diff
@@ -0,0 +1,15 @@
+{{$app := .AppName | kebabcase}}
+apiVersion: xl-deploy/v1
+kind: Applications
+spec:
+- directory: Applications/{{$app}}
+  children:
+  - name: {{$app}}
+    type: udm.Application
+    children:
+    - name: '{{.AppVersion}}'
+      type: udm.DeploymentPackage
+      deployables:
+      - name: {{$app}}
+        type: k8s.ResourcesFile
+        file: !file deployment.yaml
```

Comment:
Instead of providing a `deployment.yaml`, could you ask users to supply their own?

We've noticed that a lot of the blueprints we have developed now are too much "demo" and too little "starter". After the release of 8.6, we want to make them more into "starter" blueprints that help you get started with your own apps. See also this epic called [More cloud native blueprints for 9.0](https://xebialabs.atlassian.net/wiki/x/0oAXKg).

Reply:
If i do this, should I expect them to already have all the properties configured? Meaning, I shouldn't employ it as I do now, where I ask a user for the input, and then replace with go templating inside the deployment.yaml.tmpl file?

Reply:
Hmm, good point.

If the user provides their own, I would indeed expect them to provide a proper K8s YAML file. I wouldn't expect them to provide one with Go templating inside of it.

That does mean they'll have to repeat the name of the app and everything.